README.md

```diff
@@ -93,13 +95,15 @@ The different parameters are:
 * `provider.env`: environment variables attached to your namespace are injected to all your namespace functions
 * `scwToken`: Scaleway token you got in prerequisites
 * `scwProject`: Scaleway org id you got in prerequisites
+* `scwRegion`: Scaleway region in which the deployment will take place (accepted values are `fr-par`, `nl-ams`, `pl-waw`, default to `fr-par`)
```

Comment:
instead of listing accepted values, maybe add a link to developers doc ? : https://developers.scaleway.com/en/quickstart/#region-definition

Reply:
I'll just remove the list, user should already be aware of available regions